src/elements/element.line.js

```diff
@@ -45,17 +40,17 @@ module.exports = Element.extend({
 		ctx.save();
 
 		// Stroke Line Options
-		ctx.lineCap = vm.borderCapStyle || globalOptionLineElements.borderCapStyle;
+		ctx.lineCap = valueOrDefault(vm.borderCapStyle, globalOptionLineElements.borderCapStyle);
 
 		// IE 9 and 10 do not support line dash
 		if (ctx.setLineDash) {
-			ctx.setLineDash(vm.borderDash || globalOptionLineElements.borderDash);
+			ctx.setLineDash(valueOrDefault(vm.borderDash, globalOptionLineElements.borderDash));
```

Comment:
Reconsidering this; many additional function calls for each data point. I think we should keep using `||` in `draw`, because these are performance critical functions.

Reply:
Removed the fallbacks entirely